decompiler/pipeline/ssa/variable_renaming.py

```diff
@@ -334,3 +338,93 @@ def _classes_of(self, neighborhood: Iterable[Variable]) -> Iterable[Variable]:
         for neighbor in neighborhood:
             if neighbor in self._variable_classes_handler.color_class_of:
                 yield self._variable_classes_handler.color_class_of[neighbor]
+
+
+class ConditionalVariableRenamer(VariableRenamer):
+    """
+    A minimal renaming strategy, that renames the SSA-variables such that the total number of non SSA-variables is (almost) minimal.
+    Therefore, we construct color-classes by using lexicographical BFS on the interference graph. When the interference graph is chordal
+    this leads to a minimum number of possible variables.
```

Comment:
Update the description of this strategy, something like the following:
```suggestion
    A renaming strategy, that renames the SSA-variables such that only variables that have a relation with each other can get the same name.
    Therefore, we construct a dependency-graph with weights, telling us how likely these two variables are the same variable, i.e., copy-assignmens are more likely to be identically than compilcated computations.
```

Reply:
Fixed with 91e5db23.